### ISSUE TITLE
class weights of cityscapes

### ISSUE THREAD

**Issue by Chenfeng1271** [fregu856/deeplabv3#4]
I don't find the weight file 'class_weights.pkl',can you tell me have to get it

**Comment by fregu856** [comment]
Hi, 

You'll have to run utils/preprocess_data.py.

//
Fredrik

**Comment by Chenfeng1271** [comment]
Thank u, but I get some new troubles when I  train this code directly. It occur 'RuntimeError('cuda runtime error (77) : an illegal memory access was encountered '. So I add 'torch.cuda.set_device(1)' on the top, but here's a new one 'RuntimeError('cuda runtime error (4) : unspecified launch failure at ' These all happen at `batch_losses.append(loss_value)` in train.py. I try to skip it and sth correspond to it, but the same thing happens in `loss.backward()`. Could you give me some advice?

**Comment by ShristiDasBiswas** [comment]
> Hi,
> 
> You'll have to run utils/preprocess_data.py.
> 
> // Fredrik
Hi,

The eval code for generating IOU values is missing. Kindly push that code too.
